src/agoradatatools/great_expectations/gx/json_schemas/team_info/members_schema.json

```diff
@@ -0,0 +1,27 @@
+{
+    "$schema": "https://json-schema.org/draft/2019-09/schema",
+    "$id": "https://github.com/Sage-Bionetworks/agora-data-tools/src/agoradatatools/great_expectations/gx/json_schemas/team_info/members_schema.json",
+    "type": "array",
+    "default": [],
+    "title": "Root Schema",
+    "items": {
+        "type": "object",
+        "required": [
+            "isprimaryinvestigator",
+            "name",
+            "url"
+        ],
+        "properties": {
+            "isprimaryinvestigator": {
+                "type": "boolean"
```

Comment:
Any way to check that this field isn't null?

Reply:
From my understanding, the type field enforces that the value cannot be `null`. To allow `null` values, you would have to do something like [this](https://stackoverflow.com/questions/16241333/specify-a-value-can-be-a-string-or-null-with-json-schema).

Reply:
Ok cool, thanks.